src/client/Microsoft.Identity.Client/Utils/StopWatchService.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.Utils
+{
+    /// <summary>
+    /// Singleton timer used to measure the duration tasks.
+    /// </summary>
+    internal static class StopWatchService
+    {
+        /// <summary>
+        /// Singleton stopwatch.
+        /// </summary>
+        internal static readonly Stopwatch Watch = Stopwatch.StartNew();
```

Comment:
Why internal?

Reply:
just seems like something that should be internal. Do we want to expose it for testers?

Reply:
You're the only tester likely to test this code. If you've written unit tests that are able to avoid making this internal, then keep it private.

Reply:
oh yeah, this should be private, I misread this and thought it was referring to the class object.